content/docs/command-reference/run.md

```diff
@@ -23,8 +23,11 @@ positional arguments:
 
 `dvc run` is a helper for creating or updating
 [pipeline](/doc/command-reference/dag) stages in a `dvc.yaml` file (located in
-the current working directory). _Stages_ represent individual data processes,
-including their input and resulting outputs.
+the current working directory).
+
+_Stages_ represent individual data processes, including their input and
+resulting outputs. Combine stages to capture simple data workflows, organize
```

Comment:
For consistency with existing passive voice (typically used in cmd refs except examples sometimes).
```suggestion
resulting outputs. They can be combined to capture simple data workflows, organize
```

Reply:
Needs formatting though.